pkg/pgmodel/sql_ingest.go

```diff
@@ -711,6 +753,7 @@ func (h *insertHandler) setSeriesIds(sampleInfos []samplesInfo) (string, error)
 		}
 
 		batch.Queue("BEGIN;")
+		batch.Queue(getEpochSQL)
```

Comment:
Do we need this once per series? or just once per call? It seems the latter since the epoch can't go down?

Reply:
we probably don't, but I don't really want to make the Id-reading code more complicated right now

Reply:
I think this is pretty bad performance wise. I'd rather get this fixed now